src/components/Blocking/ReportDomain.js

```diff
@@ -61,7 +61,7 @@ const ReportDomain = React.forwardRef((props, ref) => {
     <View>
       <RBSheet
         ref={ref}
-        closeOnDragDown={false}
+        closeOnDragDown={true}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `closeOnDragDown={false}` to `closeOnDragDown={true}` will allow the user to close the RBSheet by dragging it down. Ensure this behavior is intended and aligns with your UX design.

```diff
-        closeOnDragDown={false}
+        closeOnDragDown={true}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->